### DISCUSSION TITLE
Bootstrap v5.2.x tooling setup fails

### DISCUSSION THREAD

**Discussion by victorolweny** [twbs/bootstrap#40670] [Q&A] [answered]
I have the following scripts in a `package.json` file
```js
"scripts": {
    "download": "curl -L https://github.com/twbs/bootstrap/archive/v5.2.3.tar.gz | tar xvfz -",
    "postdownload": "cd bootstrap-5.2.3 && npm install",
    "clean": "rm -rf bootstrap-5.2.3",
    "build": "cd bootstrap-5.2.3 && npm run dist",
    "watch": "cd bootstrap-5.2.3 && npm run watch"
  }
```

I am trying to follow the procedure [here](https://getbootstrap.com/docs/5.2/getting-started/contribute/) to try and customize a Bootstrap build. However, I get the following error when I run the script `npm run download` and the `postdownload`starts
```js
> postdownload
> cd bootstrap-5.2.3 && npm install

npm error code 127
npm error path .../bootstrap-5.2.3/node_modules/hugo-bin
npm error command failed
npm error command sh -c rimraf vendor && node lib/install.js
npm error sh: rimraf: command not found
npm error A complete log of this run can be found in: ..
```

I am running:
* Node v22.5.1
* npm v10.8.2
* MacOS v14.5 (Sonoma) - Intel i9

I have previously tried with `bootstrap-5.3.3` and installs successfully but I run into another issue when I run the script `npm run build`
with the error `sh: npm-run-all: command not found`. So I decided to use a version with a supported build tool (v5.2.x) but still get the error as described earlier.

Thanks

**Comment by julien-deramond** [comment]
Hello @victorolweny 

My 2 cents: if you want to contribute to Bootstrap, it might be easier to clone the repository rather than doing it like that.

Back to the question, I've just tried the scripts in the description and I don't have any issues after running `npm run download` and then `npm run postdownload` with the following configuration:

* Node v20.15.0
* npm 10.7.0
* macOS Sonoma 14.5

I tried with Node 22.5.1 and your configuration, and it works too on my machine.

It looks like you're having an issue with the `npm i` when running it in `bootstrap-5.2.3/`. Could be a network (configuration) issue or a npm configuration issue (with or without proxies, VPN, etc.).

I'm saying that because `rimraf` is actually a peer dependency that is installed by `npm i` (should be in `bootstrap-5.2.3/node_modules/.bin/rimraf`). So if there are download/network errors for some packages while running `npm i`, it could cause that.

If you go to `bootstrap-5.2.3/` and run manually `npm i`, do you have the same error?

**Reply by victorolweny** [reply]
> If you go to bootstrap-5.2.3/ and run manually npm i, do you have the same error?

Yes, I still get the same error, I have tried using another ISP still the same.

**Reply by victorolweny** [reply, marked as answer]
I think I found the issue. It looks like it is something related to location I am running the script - my path. When I run in different location whose path is defined well, all runs successful. 

Thanks @julien-deramond for the quick response.
